developer/src/kmc-kmn/src/compiler/compiler.ts

```diff
@@ -47,7 +47,7 @@ const baseOptions: CompilerOptions = {
  */
 let callbackProcIdentifier = 0;
 
-export class Compiler {
+export class KmnCompiler {
```

Comment:
missed this one…

Reply:
> missed this one…

As in, you missed it, or me? I'm not clear if there's something for me to do here?